probe-rs/src/probe/stlink/mod.rs

```diff
@@ -1482,30 +1358,95 @@ where
 {
     type Error = DebugProbeError;
 
-    fn read_ap_register(&mut self, port: impl Into<AP>, register: R) -> Result<R, Self::Error> {
-        self.read_ap_register(port, register)
+    /// Read the given register `R` of the given `AP`, where the read register value is wrapped in
+    /// the given `register` parameter.
+    fn read_ap_register(&mut self, port: impl Into<AP>, _register: R) -> Result<R, Self::Error> {
+        log::debug!("Reading register {}", R::NAME);
```

Comment:
Put this on trace? :)

Reply:
This is unchanged from before, and also done like this in the other ARM probes.